index.html

```diff
@@ -110,31 +140,36 @@ <h2 class="title">Multi-Post Stories</h2>
                 <div><button class="primary-button">See Project</button></div>
             </div>
         </div>
-     </section>
+    </section>
 
-     <!--About me Section-->
+    <!--About me Section-->
     <section class="about-me">
         <div class="personal-info">
             <h2>About Myself</h2>
-            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
             <h4 class="blue text">LET’S CONNECT</h4>
             <div>
                 <ul class="social-media">
-                    <li><a href="https://twitter.com/TarikBouari"><img class="icon" src="images/Vector2.png" alt="twitter"></a></li>
-                    <li><a href="https://www.linkedin.com/in/tarik-bouari-44b7191a6/"><img class="icon" src="images/Linkedin.png" alt="twitter"></a></li>
+                    <li><a href="https://twitter.com/TarikBouari"><img class="icon" src="images/Vector2.png"
+                                alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/tarik-bouari-44b7191a6/"><img class="icon"
+                                src="images/Linkedin.png" alt="twitter"></a></li>
                     <li> <a href="#"><img class="icon" src="images/VectorM.png" alt="twitter"></a></li>
-                    <li> <a href="https://github.com/tarikbouari"><img class="icon" src="images/Vector3.png" alt="twitter"></a></li>
+                    <li> <a href="https://github.com/tarikbouari"><img class="icon" src="images/Vector3.png"
+                                alt="twitter"></a></li>
                     <li><a href="#"><img class="icon" src="images/Vector5.png" alt="twitter"></a></li>
-                </ul>    
+                </ul>
             </div>
             <div><button class="primary-button">Get my resume</button></div>
         </div>
         <div class="code-lang">
-            <ul>
+            <ul class="ul-lang">
                 <li>
                     <div>
                         <div class="lang">
-                            
+
                             <h3>Languages</h3>
                             <img src="images/angle-down.svg" class="arrow icon" alt="arrow-down">
                         </div>
```

Comment:
- [ ]  Your about section should look similar to the provided Figma design. 
![image](https://user-images.githubusercontent.com/43877689/177634975-cd6f27da-b376-491e-8d5f-8632c82cacb1.png)